rc/.eslintrc

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": ["standard", "standard-react"]
+  "extends": ["standard", "standard-react"],
+  "rules": {
+    "object-curly-even-spacing": [2, "either"]
```

Comment:
Not really able to add this custom rule to the public configuration plugins, for obvious reasons.....may be easier when these custom rules become their own plugin at some point in the future.